### PR TITLE
Style Improvements for Sort.js

### PR DESCRIPTION
**Pull request by jbts**
Working on #1601 for `src/utils/Sort.js`.

Most of the changes I've made are small, like replacing `==` with `===` or `var` with `let`.
Some changes are a little more involved, like renaming a variable to use camelCase or removing an unused parameter from a function.

Ideally these changes just affect the style of the code and not the code execution/logic.

There are a couple tricky style issues left that I'm not sure how to handle. I'll see if I can highlight them with some comments.

### CODE REVIEW

File: src/utils/Sort.js [hunk 1]
```diff
@@ -645,158 +679,179 @@ export function cardGetLabels(card, sort) {
...
+    ];
+  }
+  if (sort === 'Artist') {
     return [card.details.artist];
-  } else if (sort == 'Legality') {
+  }
+  if (sort === 'Legality') {
     return Object.entries(card.details.legalities)
       .filter(([k, v]) => ['legal', 'banned'].includes(v))
       .map(([k, v]) => k);
```
Comment (jbts) [inline src/utils/Sort.js:700]: Here one of the functions doesn't use the key `k` and the other doesn't use the value `v`.

**Comment by jbts** [comment]
I can't seem to comment in the same way for the other style issue, but it's on line 906. The first argument `label` isn't used by the function.

File: src/utils/Sort.js (hunk 1 above)
Comment (jbts) [reply src/utils/Sort.js:700]: Alright. I'll plan to add a comment to tell eslint to ignore those errors then.

Comment (dekkerglen) [reply src/utils/Sort.js]: You can replace the variable name with `_` and it will suppress this error iirc

Comment (jbts) [reply src/utils/Sort.js]: That hasn't been working for me, still getting the same error with `_` and no comment.

**Pull request merged**